inc/sainexthopgroup.h

```diff
@@ -314,6 +349,33 @@ typedef enum _sai_next_hop_group_member_attr_t
      */
     SAI_NEXT_HOP_GROUP_MEMBER_ATTR_COUNTER_ID,
 
+    /**
+     * @brief Quality threshold for least cost ARS paths. Crossing down the threshold will result in using the non least cost sub optimal path.
+     *
+     * @type sai_uint32_t
+     * @flags CREATE_AND_SET
+     * @default 16
+     */
+    SAI_NEXT_HOP_GROUP_MEMBER_ATTR_ARS_PRIMARY_PATH_QUALITY_THRESHOLD,
```

Comment:
vendor-specific

Reply:
This has nothing to do with algorithm but is a configuration for non least cost path selection.